meta-lmp-base/recipes-connectivity/bluez5/files/0001-build-add-initial-support-for-building-with-libedit.patch

```diff
@@ -12,44 +12,42 @@ Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
 Signed-off-by: Jose Quaresma <jose.quaresma@foundries.io>
 
 ---
- Makefile.am                   |  2 +-
- Makefile.tools                | 18 +++++++++---------
+ Makefile.am                   |  1 +
+ Makefile.tools                | 17 +++++++++--------
  attrib/interactive.c          | 10 ++++++++++
  client/display.c              | 15 +++++++++++++++
  configure.ac                  |  2 ++
  src/shared/shell.c            | 26 ++++++++++++++++++++++++++
  tools/mesh-gatt/onoff-model.c |  6 ++++++
  tools/mesh-gatt/prov.c        |  5 +++++
  tools/obex-client-tool.c      |  4 ++++
- 9 files changed, 78 insertions(+), 10 deletions(-)
+ 9 files changed, 78 insertions(+), 8 deletions(-)
 
 diff --git a/Makefile.am b/Makefile.am
-index 44267d0bf..24cc451cb 100644
+index 081d0aa050..9e019f3f82 100644
 --- a/Makefile.am
 +++ b/Makefile.am
-@@ -25,7 +25,7 @@ pkgincludedir = $(includedir)/bluetooth
+@@ -25,6 +25,7 @@ pkgincludedir = $(includedir)/bluetooth
  pkginclude_HEADERS =
  
  AM_CFLAGS = $(MISC_CFLAGS) $(WARNING_CFLAGS) $(UDEV_CFLAGS) $(LIBEBOOK_CFLAGS) \
--				$(LIBEDATASERVER_CFLAGS) $(ell_cflags)
-+				$(LIBEDATASERVER_CFLAGS) $(READLINE_CFLAGS) $(ell_cflags)
++				$(READLINE_CFLAGS) \
+ 				$(LIBEDATASERVER_CFLAGS) $(ell_cflags)
  AM_LDFLAGS = $(MISC_LDFLAGS)
  
- confdir = $(sysconfdir)/bluetooth
 diff --git a/Makefile.tools b/Makefile.tools
-index df4cad065..68ffad731 100644
+index 0bc0f232da..a957653f24 100644
 --- a/Makefile.tools
 +++ b/Makefile.tools
-@@ -14,7 +14,7 @@ client_bluetoothctl_SOURCES = client/main.c \
- 					client/admin.h client/admin.c \
- 					client/player.h client/player.c
- client_bluetoothctl_LDADD = gdbus/libgdbus-internal.la src/libshared-glib.la \
--				$(GLIB_LIBS) $(DBUS_LIBS) -lreadline
-+				$(GLIB_LIBS) $(DBUS_LIBS) $(READLINE_LIBS)
+@@ -16,6 +16,7 @@ client_bluetoothctl_SOURCES = client/main.c \
+ 					client/mgmt.h client/mgmt.c
+ client_bluetoothctl_LDADD = lib/libbluetooth-internal.la \
+ 			gdbus/libgdbus-internal.la src/libshared-glib.la \
++			$(READLINE_LIBS) \
+ 			$(GLIB_LIBS) $(DBUS_LIBS) -lreadline
```

Comment:
New version will have READLINE_LIBS and -lreadline?

Reply:
good catch.
the rebase was not done well, fixed now.
thanks